<launch>

<arg name="name" default="whycon"/>
<arg name="targets" default="1"/>
<arg name="outer_diameter" default=".55"/>
<arg name="inner_diameter" default=".20"/>
<arg name="input_queue_size" default="1000"/>

<node name="whycon" type="whycon" pkg="whycon" output="screen">
  <param name="targets" value="$(arg targets)"/>
  <param name="outer_diameter" value="$(arg outer_diameter)"/>
  <param name="inner_diameter" value="$(arg inner_diameter)"/> 
  <param name="input_queue_size" value="$(arg input_queue_size)"/> 
  <param name="name" value="$(arg name)"/>

  <remap to="/usb_cam/camera_info" from="/camera/camera_info" />
  <remap to="/usb_cam/image_rect_color" from="/camera/image_rect_color" />
    
</node>

<!-- Uncomment the following when you want to run task 1.1 -->
<node name="task_3_pos_hold" type="pos_hold.py" pkg="pollinator_bee" output="screen"/> 

<!--Uncomment the following when you want to run task 1.2 -->
<node name="task_3_ip" type="ip.py" pkg="pollinator_bee" output="screen"/>

<!-- PID tune node -->
<!--<node name="pid_tune_gui" type="pid_tune_gui.py" pkg="pid_tune" output="screen"/> -->

<!-- PlotJuggler node -->
<!-- <node name="plotJuggler" type="PlotJuggler" pkg="plotjuggler" output="screen"/> -->


<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
  <remap from="image" to="/whycon/image_out"/>
  <param name="autosize" value="true" />
</node>

<node name="image1" pkg="image_view" type="image_view" respawn="false" output="screen">
  <remap from="image" to="/imageout"/>
  <param name="autosize" value="true" />
</node>















</launch>
